src/richie/apps/core/context_processors.py

```diff
@@ -33,46 +34,65 @@ def site_metas(request):
             "domain": site_current.domain,
             "web_url": f"{protocol:s}://{site_current.domain:s}",
         },
-        "AUTHENTICATION": {
-            "PROFILE_URLS": json.dumps(
-                [
-                    {
-                        "label": str(url["label"]),
-                        "action": str(
-                            url["href"].format(
-                                base_url=authentication_delegation["BASE_URL"]
-                            )
-                        ),
-                    }
-                    for url in authentication_delegation["PROFILE_URLS"]
-                ]
-            ),
+        "FRONTEND_CONTEXT": {
+            "context": {
+                "csrftoken": get_token(request),
+                "environment": getattr(settings, "ENVIRONMENT", ""),
+                "release": getattr(settings, "RELEASE", ""),
+                "sentry_dsn": getattr(settings, "SENTRY_DSN", ""),
+            }
         },
-        "FRONTEND_CONTEXT": json.dumps(
+    }
+
+    if getattr(settings, "CDN_DOMAIN", False):
+        context["CDN_DOMAIN"] = settings.CDN_DOMAIN
+
+    if getattr(settings, "AUTHENTICATION_DELEGATION", False):
+        context.update(
             {
-                "context": {
-                    "csrftoken": get_token(request),
-                    "environment": getattr(settings, "ENVIRONMENT", ""),
-                    "release": getattr(settings, "RELEASE", ""),
-                    "sentry_dsn": getattr(settings, "SENTRY_DSN", ""),
-                    "authentication": {
-                        "endpoint": authentication_delegation["BASE_URL"],
-                        "backend": authentication_delegation["BACKEND"],
-                    },
-                    "lms_backends": [
-                        {
-                            "endpoint": lms["BASE_URL"],
-                            "backend": lms["BACKEND"],
-                            "course_regexp": lms["JS_COURSE_REGEX"],
-                            "selector_regexp": lms["JS_SELECTOR_REGEX"],
-                        }
-                        for lms in getattr(settings, "LMS_BACKENDS", [])
-                    ],
+                "AUTHENTICATION": {
+                    "PROFILE_URLS": json.dumps(
```

Comment:
Make only first level keys lower case (the ones that will available as variables in the template...
```suggestion
                    "profile_urls": json.dumps(
```